command/genesis/polybft_params.go

```diff
@@ -222,9 +227,17 @@ func (p *genesisParams) deployContracts() (map[types.Address]*chain.GenesisAccou
 			return nil, err
 		}
 
-		allocations[contract.address] = &chain.GenesisAccount{
-			Balance: big.NewInt(0),
-			Code:    artifact.DeployedBytecode,
+		// ChildValidatorSet must have funds pre-allocated, because of withdrawal workflow
+		if contract.name == "ChildValidatorSet" {
```

Comment:
Can you move this outside the for loop? You can simply get the `ChildValidatorSet` from the allocations map and update it's `Balance`.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1112/commits/a2d098ad34c0f9f9997ae52ba249afce31833643